src/components/PrintModal/PrintModal.js

```diff
@@ -557,6 +559,16 @@ class PrintModal extends React.PureComponent {
                       disabled={isPrinting}
                       center
                     />
+                    <Choice
+                      dataElement="annotationsPrintOption"
+                      id="include-annotations"
+                      name="annotations"
+                      label={t('option.print.includeAnnotations')}
+                      disabled={isPrinting}
+                      onChange = {() => this.setState(state => ({ includeAnnotations: !state.includeAnnotations }))}
```

Comment:
Hmm do you think to make the code look consistent we should make it an uncontrolled component like other `<Choice>` and use `ref.current.checked` instead? I'm ok with this.

Reply:
I am not able to use the `ref.current.checked` if it needs to be `checked` by default. Maybe its because it warps in the <Choice /> component? [react-toolkit Choice component](https://pdftron.github.io/webviewer-react-toolkit/?path=/story/components-choice--controlled)

Reply:
I see. I think the rest of the props aren't documented. I checked the code and it seems that it will pass through most of the props to the <input> element.

If you look at the first `Choice` in this file it takes in a `defaultChecked` prop. I think you can try it and see if it works?

Reply:
Sure, I will try it. Thanks!

Reply:
The `defaultChecked` seems not working for the checkbox. @liamross

Reply:
ok I see. Let's keep it this way then.